### PR TITLE
fedora/centos: Add a build env setup script

### PR DESCRIPTION
**Pull request by baalajimaestro**
Let buildbots use fedora now

### CODE REVIEW

**Comment by baalajimaestro** [comment]
Shellcheck has failed on a file from the master branch

File: setup/fedora-centos.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    libxml2 \
+                    libxml2-devel.* \
+                    libxslt \
+                    zlib.* \
+                    lzma \
+                    xz
+
+
+# What if you didnt set this up?
+GIT_USERNAME="$(git config --get user.name)"
```
Comment (akhilnarang) [inline setup/fedora-centos.sh]: USERNAME to NAME

**Comment by baalajimaestro** [comment]
Fedora has too many issues to face as a buildsystem. I would not recommend using it for aosp buildsystems.

**Comment by gotenksIN** [comment]
s/to f*//

**Pull request closed without merging**